app/src/main/res/values/strings.xml

```diff
@@ -187,6 +187,9 @@
     <string name="tx_confirmations_execute_waiting">Execute (%1$d more confirmation needed)</string>
     <string name="tx_confirmations_executed">Executed</string>
 
+    <string name="signing_owner_dialog_remove">Remove</string>
+    <string name="signing_owner_dialog_cancel">Cancel</string>
+    <string name="signing_owner_dialog_description">Removing the owner key only removes it from this app. It doesn’t delete any Safes from this app or from blockchain. Safes controlled by this key will no longer be able to sign transactions in this app.</string>
```

Comment:
I find the last sentence unclear. "Safes [...] will no longer be able to sign transactions [...]" Since when do safes sign transactions. I assume this is the specified wording?

@tschubotz

Reply:
yea, seem like that's the current wording from the designs (https://zpl.io/VYe4pXM). Agree, it's not 100% technically correct.

What would you write instead?

How about using the following last sentence instead? 

`For Safes controlled by this owner key, you will no longer be able to sign transactions in this app.`

cc @posthnikova

Reply:
@tschubotz @biafra23 Agree.